src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.html

```diff
@@ -0,0 +1,62 @@
+<div
+  *ngIf="selectedTab == Tabs.clusters"
+  i18n
+>
+  <h1>Clusters</h1>
+  <h5>Logical management units that may map to one or more managed Samba service</h5>
```

Comment:
localization should be on the closest tags instead of parent for better string extraction. Same for below and other places.
```suggestion
  <h1 i18n>Clusters</h1>
  <h5 i18n>Logical management units that may map to one or more managed Samba service</h5>
```

Reply:
Oh okay, I didn't know. I have updated it